nbs/12_gsa_gls/15-preprocess_gene_expr_correlations.ipynb

```diff
@@ -1908,7 +1908,7 @@
   },
   {
    "cell_type": "markdown",
-   "id": "offensive-earthquake",
+   "id": "transsexual-liver",
```

Comment:
I'm not sure which package is generating these but seems an odd choice for them to include adjectives associated with marginalized groups! (Especially since I think this term is kind of uncommon now)

Reply:
Jupyter Notebook is automatically generating these names, and [apparently](https://stackoverflow.com/questions/66678305/where-does-the-id-field-eg-wound-transition-in-a-jupyter-notebook-come-from-a), the idea is to have "human-readable cell identifiers". Many people have complained about this, and seems there is no way to change it :( But it seems [you can filter those IDs out](https://stackoverflow.com/a/68037340), so I will do that in the next project.

I removed these weird cell identifiers from these notebooks.

Reply:
So weird!!! Jupyter, why!